pkg/spec/createconfig.go

```diff
@@ -406,7 +415,15 @@ func (c *CreateConfig) GetContainerCreateOptions(runtime *libpod.Runtime) ([]lib
 
 		options = append(options, libpod.WithIPCNSFrom(connectedCtr))
 	}
+	if IsPod(string(c.IpcMode)) {
+		options = append(options, libpod.WithIPCNSFromPod())
+	}
+
+	if IsPod(string(c.UtsMode)) {
+		options = append(options, libpod.WithUTSNSFromPod())
+	}
```

Comment:
Maybe I missed it, but I don't see a lock on the container at this point.  This WithUTSNSFromPod() checks to see if the container is a member of a pod or not.  Is it possible that another process could change that membership after this check and before this process here finishes?  Would that be problematic?

Reply:
Pod membership never changes after a container is created, so we should be fine as long as the pod and/or infra container in question aren't removed while we're making the spec. That should also be fine, but errors might be a little less than optimal (we'd get complaints from libpod that we were trying to depend on a nonexistant container, and be added to a nonexistant pod)